test/test_connectionpool.py

```diff
@@ -32,6 +32,26 @@
 from dummyserver.server import DEFAULT_CA
 
 
+class HTTPUnixConnection(HTTPConnection):
+    def __init__(self, host, timeout=60, **kwargs):
+        super(HTTPUnixConnection, self).__init__('localhost')
+        self.unix_socket = host
+        self.timeout = timeout
+        self.sock = None
+
+    # unused
```

Comment:
This doesn't look like it's meant to go here?

Reply:
I commented out these lines because of flake8 errors in travis. I can remove commented lines.

But classes `HTTPUnixConnection*` are used in tests `test.*_same_host_custom_protocol`
Or maybe I misunderstood comments

Reply:
Commenting out the block has no effect, nothing gets overwritten, so you should remove it.